lib/susedistribution.pm

```diff
@@ -740,6 +740,14 @@ sub activate_console {
         assert_screen("text-logged-in-$user", 60);
         $self->set_standard_prompt($user);
     }
+    elsif ($console eq 'powerhmc-ssh') {
+        assert_screen "password-prompt-hmc";
+        type_password get_required_var('HMC_PASSWORD');
+        send_key('ret');
+        my $user = get_var('HMC_USERNAME', 'hscroot');
+        assert_screen("text-logged-in-$user", 60);
+        $self->set_standard_prompt($user);
+    }
```

Comment:
ok, before we introduce this can we now please work on something like https://github.com/os-autoinst/os-autoinst/compare/master...okurz:enhance/dry_backend to not repeat all arguments? Would be nice if the code would look like this:

```perl
elsif ($console eq 'hypervisor-ssh') {
        assert_screen "password-prompt-hypervisor";
        type_password get_required_var('HYPERVISOR_PASSWORD');
        send_key('ret');
        my $user = get_var('HYPERVISOR_USERNAME', 'root');
        assert_screen("text-logged-in-$user", 60);
        $self->set_standard_prompt($user);
    }
```